.github/workflows/autopep8.yml

```
@@ -0,0 +1,23 @@
+name: autopep8
+on: pull_request
+jobs:
+  autopep8:
```

```suggestion
  format_python:
```
単に好みの問題だけど、htmlとかのフォーマットのジョブもいづれは追加することを考えて...